bin.js

```diff
@@ -72,10 +72,8 @@ if (argv[0] == 'server') {
   // write RPC manifest to ~/.ssb/manifest.json
   fs.writeFileSync(manifestFile, JSON.stringify(server.getManifest(), null, 2))
 
-  if(process.stdout.isTTY && (config.logging.level != 'info'))
-    ProgressBar(server.progress)
+  if (process.stdout.isTTY && (config.logging.level !== 'info')) { ProgressBar(server.progress) }
```

Comment:
Standard allows dropping `{}` curlies as long as it's on the same line. Here I'd recommend 

```
if (process.stdout.isTTY && (config.logging.level !== 'info')) ProgressBar(server.progress)
```

as it is slightly less noise visually, or just

```
if(process.stdout.isTTY && (config.logging.level != 'info')) {
    ProgressBar(server.progress)
}
```

if you want to preserve the original shape.

Reply:
the first line would be too long now, would probably wrap anyway so the second would be preferred.

Reply:
> Standard allows dropping {} curlies as long as it's on the same line.

Yep! This change was made with `standard --fix`, but those curly braces are optional.